src/index.html

```diff
@@ -9,6 +9,34 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a class="logo-img" href="#"><img src="./images/logo.png" alt="logo"></a>
+      <nav>
+        <ul class="item-nav">
+          <li class="item">
+            <a class="link is-active" href="#apple">apple</a>
+          </li>
+          <li class="item">
+            <a class="link" href="#samsung">samsung</a>
+          </li>
```

Comment:
```suggestion
        <ul class="item-nav">
          <li class="item">
            <a class="link is-active" href="#apple">apple</a>
          </li>
          
          <li class="item">
            <a class="link" href="#samsung">samsung</a>
          </li>
```
fix code style all over the page